contracts/mocks/modules/StakingPool/StakingPoolMockCover.sol

```diff
@@ -2,45 +2,44 @@
 
 pragma solidity ^0.8.18;
 
-import "../../interfaces/IStakingPool.sol";
-import "../../interfaces/ICover.sol";
+import "../../../interfaces/IStakingPool.sol";
+import "../../../interfaces/ICover.sol";
+import "../../generic/CoverGeneric.sol";
 
-contract StakingPoolMockCover {
+contract StakingPoolMockCover is CoverGeneric {
```

Comment:
SPMockCover

Reply:
We don't have a shortcode for staking pool, that's why it was left like this. SP is already used by StakingProducts, I suggest maybe using ST as shortcode at least in the tests, open to suggestions of course.

Reply:
that works imo